src/i965_drv_video.h

```diff
@@ -69,8 +69,8 @@
 #define DEFAULT_SATURATION      50
 
 #define ENCODER_QUALITY_RANGE     2
-#define ENCODER_QUALITY_RANGE_AVC     8
-#define ENCODER_QUALITY_RANGE_HEVC    8
+#define ENCODER_QUALITY_RANGE_AVC    7
+#define ENCODER_QUALITY_RANGE_HEVC   7
 #define ENCODER_DEFAULT_QUALITY   1
```

Comment:
for ENCODER_QUALITY_RANGE, the actual max input quality level is ENCODER_QUALITY_RANGE - 1. but this update let ENCODER_QUALITY_RANGE_AVC and ENCODER_QUALITY_RANGE_HEVC break this rule.